### PR TITLE
Enable flags again

### PR DESCRIPTION
**Pull request by Strappazzon**
#825 removed national flags representing languages, but this also removed flags representing **countries** (see screenshot below). I added the flags again but without representing languages.

### CODE REVIEW

File: assets/img/flags/.move svg files in the 4x3 folder if you want to use them [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Mikaela) [inline assets/img/flags/.move svg files in the 4x3 folder if you want to use them]: I think this is a bit of nasty filename, could the information be in README.md of that directory instead?

Comment (JonahAragon) [reply assets/img/flags/.move svg files in the 4x3 folder if you want to use them]: Agree, this file doesn't appear necessary.

**Comment by Strappazzon** [comment]
@JonahAragon The `4x3` folder was there before the removal.
I removed `4x3`, `.move svg files` and edited `flag-icon.scss` accordingly in https://github.com/privacytoolsIO/privacytools.io/pull/873/commits/8be2c18416d29bb7796151dd8ee52436c059bfa1.

**Comment by Mikaela** [comment]
> This seems fine to me

I am taking this as OK to merge as I don't see anything strange anymore either.

Thank you for your contribution @Strappazzon :)

**Pull request merged**